lib/model/schema_versions.dart

```diff
@@ -0,0 +1,238 @@
+import 'package:drift/internal/versioned_schema.dart' as i0;
+import 'package:drift/drift.dart' as i1;
+import 'package:drift/drift.dart'; // ignore_for_file: type=lint,unused_import
+
+// GENERATED BY drift_dev, DO NOT MODIFY.
```

Comment:
Let's give this a `.g.dart` name, so it's clear in a more uniform way that it's generated.